src/app/projects/create-project/create-project.styl

```diff
@@ -87,6 +87,24 @@ md-select.create-project-select
   text-align center
   line-height 40px
 
+.create-project-panel-progress-item-number-in-progress
+  animation: pulse 5s infinite;
+
+@keyframes pulse {
+  0% {
+    background-color white
+    color black
```

Comment:
it's not required, styl files are then transformed into css (with the expected format)
you can cleanup all : and ;